core/insertion_marker_manager.ts

```diff
@@ -303,15 +303,9 @@ export class InsertionMarkerManager {
     if (lastOnStack && lastOnStack !== this.topBlock.nextConnection) {
       available.push(lastOnStack);
       this.lastOnStack = lastOnStack;
-      if (this.lastMarker) {
-        eventUtils.disable();
-        try {
-          this.lastMarker.dispose();
-        } finally {
-          eventUtils.enable();
-        }
+      if (!this.lastMarker) {
```

Comment:
Instead of doing this here, can you move it to the constructor & update the JSDoc for this function?

Like:
```
this.firstMarker = /* etc */;
this.initAvailableConnections();
if (this.lastOnStack) this.lastMarker = this.createMarkerBlock(/* etc */);
```

Reply:
Done.